src/fstests/README.md

```diff
@@ -148,6 +149,34 @@ tarball is available to test.  This will then generate a job to automatically
 download the tarball, extract it, build a kernel with the config required for
 fstests, run the smoke tests, parse the results and send it to the API.
 
-> **Warning** This needs to be implemented as part of this PoC!  See the
-> [GitHub workboard](https://github.com/orgs/kernelci/projects/15/views/4) for
-> more details about the current status.
+To set up a local instance you first need to set instances of [Kernel CI API](https://kernelci.org/docs/api/getting-started/#setting-up-an-api-instance) and [Kernel CI Pipeline](https://kernelci.org/docs/api/getting-started/#setting-up-a-pipeline-instance) in your local environment. Once you are able to interact with the API with cURL or any tool of your preference and have also set up the pipeline, now it is time to start `fstests runner.py`.
+
+Before you start running the script you need to get base templates from [Kernel CI core](https://github.com/kernelci/kernelci-core/), especifically, you will need `python.jinja2` and `shell-python.jinja2` templates. The easiest way to do this is to install Kernel CI core following the steps below:  
+
+```bash
+git clone https://github.com/kernelci/kernelci-core.git
+cd kernelci-core
+pip3 install -r requirements-dev.txt
+python3 setup.py install
+sudo cp -R config /etc/kernelci
+```
+
+After that you will need to set the configuration file proper, for now, we just need to set 3 variables for `fstests runner` :
+
+- `output`: Path to a directory that could be used as during runtime as tmp.
+- `xfstests_bld_path`: Path to the directory where `xfstests` was built.
+- `db_config`:  Database location. You should be able to use a pre-set token to access this DB. 
+
+```bash
```

Comment:
Are you sure \`\`\`bash works?  I think the standard mode for shell script syntax highlighting is \`\`\`sh

Reply:
yes, I am! It is an alias for `shell`, as `sh` :) in [`gfm`](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6045)

Reply:
Right, I think there are different variants and implementations of Markdown.  That seems to be fine for GitHub, and the standard seems to be `shell` like you wrote.  So anyway, that's all fine :)

Reply:
exactly there are some different flavours of markdown..